mache-chroniclemap/src/test/java/com/excelian/mache/chroniclemap/ChronicleMapMacheProvisionerShould.java

```diff
@@ -0,0 +1,88 @@
+package com.excelian.mache.chroniclemap;
+
+import com.excelian.mache.chroniclemap.ChronicleMapMacheProvisioner.ChronicleMapBuilderConfig;
+import com.excelian.mache.core.HashMapCacheLoader;
+import net.openhft.chronicle.map.ChronicleMapBuilder;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.mockito.Mockito;
+
+import java.io.File;
+import java.util.List;
+
+import static com.excelian.mache.chroniclemap.ChronicleMapMacheProvisioner.chronicleMap;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.verify;
+import static org.mockito.internal.util.reflection.Whitebox.getInternalState;
+
+public class ChronicleMapMacheProvisionerShould {
+
```

Comment:
Hmmm, has that guy evolved to be trying to do TDD? Why does he already know his method name when he's deciding on what his test should be named? We're talking SRP class behaviours right? :wink: 

I guess it's a hugely subjective/preferential thing so I think we'll be able to find plenty of articles explaining the different [rationales](http://codurance.com/2014/12/13/naming-test-classes-and-methods/) and will each have our own opinions on the weighting of the given merits.

I vote we standardise on the way I've got it cos it's the one I like haha :smiley: (I'm being facetious but secretly hoping we all say yes)